### ISSUE TITLE
Broken

### ISSUE THREAD

**Issue by Spleens1** [vJoeyz/uj_base#3]
Nothing works with this mod. I've strictly followed the tutorial and put it in the right directory of my server but this whole mod does not work at all.

**Comment by X0tabuch** [comment]
> С этим модом ничего не работает. Я строго следовал руководству и поместил его в правильный каталог моего сервера, но весь этот мод вообще не работает.

I completely agree with you, but some person told me that I need to add a line of code to the downloaded html, but I still don’t understand which specific html config is needed or what?
